doc/ExpertOptions.md

```diff
@@ -1470,58 +1480,62 @@ You would write the following manifest:
 {
   "test.input_manifest___": {
     "s": "hello",
-    "f": "dx://file-xxx"
+    "f": "project-aaa:file-xxx"
```

Comment:
do we accept `dx://` prefix in manifests?  The input files that are passed to dxCompiler for WDL and CWL do accept it (or do the require dx:// prefix?), so this inconsistency would be confusing.

Reply:
why not specify input in the same format as for the non-manifest mode, and have dxCompiler handle the translation?  .

Reply:
Up to the discussion. This was to encourage using of the fully qualified names to avoid bugs reported in APPS-1269 and APPS-1270 (until those are fixed). Please see comments to APPS-1270 to get the context, but shortly I envision the necessity of a fully qualified name in cases when file form a different project is referenced. I'll revert for now.